src/Microsoft.Diagnostics.Monitoring.RestServer/CorsConfiguration.cs

```diff
@@ -2,12 +2,12 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
 
+#if UNITTEST
+namespace Microsoft.Diagnostics.Monitoring.UnitTests.Options
```

Comment:
Why a custom namespace for unit test versions of these?

Reply:
Because these are compiled into the unit test assembly, I like to keep the root namespace the same with the assembly. It's not important to do it though.